### PR TITLE
cypress:added retries parameter in some fragile tests

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rashesh.padia@collabora.com>
Change-Id: Ia99f75302a988fc2ef744807e01ba6441ff161ec

* Target version: master

### CODE REVIEW

File: cypress_test/integration_tests/multiuser/calc/repair_document_spec.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 var helper = require('../../common/helper');
 var calcHelper = require('../../common/calc_helper');
 
-describe('Repair Document', function() {
+describe('Repair Document',function() {
```
Comment (Ashod) [inline cypress_test/integration_tests/multiuser/calc/repair_document_spec.js:6]: The space was actually better, and consistent with the other cases.

**Review by Ashod — COMMENTED** [review]
Can we do this for all tests? I don't see why we can't retry any failed test at least once, instead of re-running the whole build, which we often do to make sure it wasn't a random failure. There might be an easy way to configure the default retry count of all tests.

File: cypress_test/integration_tests/multiuser/calc/repair_document_spec.js (hunk 1 above)
Comment (eszkadev) [reply cypress_test/integration_tests/multiuser/calc/repair_document_spec.js:6]: +1, better to not change unrelated lines

**Comment by eszkadev** [comment]
Can we do this also for other "repair_document_spec.js" (writer and impress) ? I noticed it failed for me also in impress case.

**Comment by eszkadev** [comment]
@Ashod maybe let's check if it helps first before we apply it to all tests (these failures were mostly present on gerrit's Jenkins)

**Review by eszkadev — APPROVED** [review]

**Pull request merged**